apps/api/src/infrastructure/MediaLiveChannelsManager.ts

```diff
@@ -782,4 +801,146 @@ export class MediaLiveChannelsManager implements LiveChannelsManager {
       })
     );
   }
+
+  private createInputCommand(
+    type: InputType,
+    source: Source,
+    inputName: string
+  ): CreateInputCommandInput {
+    if (
+      isString(source) &&
+      (type === InputType.TS_FILE ||
+        type === InputType.MP4_FILE ||
+        type === InputType.URL_PULL)
+    ) {
+      return {
+        Name: inputName,
+        Type: type,
+        Sources: this.buildClassicSource(source),
+      };
+    }
+    if (isRtp(source, type)) {
+      return this.buildRtp(source, inputName);
+    }
+    if (isRtmpPush(source, type)) {
+      return this.buildRtmpPush(source, inputName);
+    }
+    if (isRtmpPull(source, type)) {
+      return this.buildRtmpPull(source, inputName);
+    }
+    if (isMediaConnect(source, type)) {
+      return this.buildMediaConnect(source, inputName);
+    }
+    if (isSrtCaller(source, type)) {
+      return this.buildSrtCaller(source, inputName);
+    }
+    return {
```

Comment:
Shouldn't we just throw an error here? It shouldn't be possible this reach this return statement right?